src/bin/server.rs

```diff
@@ -51,11 +62,23 @@ async fn main() -> Result<(), Error> {
 
     spawn(write_stats(dir, stats));
 
+    let local_assets = Path::new("assets");
```

Comment:
These should be injected via an environment variable for consistency with the other parameters and to avoid baking assumptions into the source. (The environment variables itself would be set by xtask for local development and in the systemd unit for deployment.)